src/plugins/drawing/model/drawing-content.ts

```diff
@@ -396,25 +397,6 @@ export const DrawingContentModel = NavigatableTileModel
           self.setSelectedIds(newIds);
         },
 
-        moveObjects(moves: DrawingObjectMove[]) {
-          moves.forEach(move => {
-            const object = self.objectMap[move.id];
-            object?.setPosition(move.destination.x, move.destination.y);
-          });
-        },
-
-        updateImageUrl(oldUrl: string, newUrl: string) {
-          if (!oldUrl || !newUrl || (oldUrl === newUrl)) return;
-          // Modify all images with this url
-          self.objects.forEach(object => {
-            if (object.type !== "image") return;
-            const image = object as ImageObjectType;
-            if (image.url === oldUrl) {
-              image.setUrl(newUrl);
-            }
-          });
-        },
```

Comment:
I investigated this and do not think that is true.  `ImageToolComponent` (in the Image tile) calls its own `ImageContentModel.updateImageUrl`.  Nothing that I can find (other than tests) calls `DrawingContentModel.updateImageUrl`.  Removing this method, I can still upload a background image to a drawing.

Reply:
I also testing copy-pasting an image in.

The relevant code in each case calls `gImageMap.getImage(url)` and creates the drawing element in the `then` clause, so it may not actually display until after the upload to Firebase.

Reply:
Sorry, yes you are right, this isn't called anywhere. 

It is possible that we've lost some behavior here in some previous commit. I'm pretty sure CLUE used to upload external images to firebase and then update the URL in the document state. But I'm not able to see that happen either in the drawing tile or in the image tile. I tried dragging in an image from another web page, for example Larry's picture here: https://concord.org/about/staff/larry-behan/ When I did this it created an image tile with the original url: https://concord.org/wp-content/uploads/2023/07/larry-behan2.png
I also tried adding this url to some authored content then dragging the image tile from the content into a student doc. When either dragging it into a drawing tile to add an image object, or when dragging into the document to make a image tile, the result was the original URL being saved in the document. 

Anyhow I think it is fine to delete the method as you've done. If we ever need this functionality back we can re-implement it.